### PR TITLE
Add keepScores to SortParams [backward compatible]

### PR DESCRIPTION
**Pull request by hugomano**
🙈don't forget to squash if you merge because there are more commits than LOCs 🙄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,10 @@ onSearch: (searchQuery) => {
 }
 ```
 
+Frecency adds and removes `_frecencyScore` attribute for compare results.
+You can output results with scores by assigning `keepScores` parameter to `true`.
```
[the remark this answers is not in the document]
Comment (alexstrat) [reply README.md]: Added in b8c11ce AFAIK

[the remark this answers is not in the document]
Comment (wearhere) [reply README.md]: Oh thanks! Didn't see that @hugomano had pushed to the PR. Wish GitHub triggered notifications on that…

**Comment by wearhere** [comment]
lgtm thanks @hugomano !

**Comment by wearhere** [comment]
published as frecency@1.2.0

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (hugomano) [reply README.md]: No worries 😃

**Pull request merged**